app/views/candidate_mailer/submit_application_email.text.erb

```diff
@@ -12,14 +12,24 @@ Your application reference is <%= @application_form.support_reference %>.
 
 You can track the progress of your <%= 'application'.pluralize(@application_form.application_choices.count) %> here:
 
-<%= candidate_interface_application_form_url %>
+<% if FeatureFlag.active?('encrypted_candidate_id') %>
+  <%= candidate_interface_application_form_url(id: @encrypted_candidate_id) %>
+<% else %>
+  <%= candidate_interface_application_form_url %>
+<% end %>
 
 # Making changes to your application
 
+<% if FeatureFlag.active?('encrypted_candidate_id') %>
+  <% sign_in_url = candidate_interface_sign_in_url(id: @encrypted_candidate_id) %>
```

Comment:
We should extract this into a helper - something like `candidate_sign_in_url(candidate)`.